mne/tests/test_coreg.py

```diff
@@ -279,3 +280,51 @@ def test_get_mni_fiducials():
     fids_est = np.array([f['r'] for f in fids_est])
     dists = np.linalg.norm(fids - fids_est, axis=-1) * 1000.  # -> mm
     assert (dists < 8).all(), dists
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('ref_scale', [
+    (None, [1., 1., 1.]),
+    ('uniform', [1., 1., 1.]),
+    ('3-axis', [1., 1., 1.]),
+    ('uniform', [0.8, 1., 1.]),
+    ('3-axis', [0.8, 0.8, 0.8])])
```

Comment:
These are reversed (the uniform should be uniform), and let's try having the Y and Z be non-unity
```suggestion
    ('uniform', [0.8, 0.8, 0.8]),
    ('3-axis', [0.8, 1.2, 1.2])])
```